### DISCUSSION TITLE
Warning: Missing Node Types : When loading the graph, the following node types were not found: BiRefNet

### DISCUSSION THREAD

**Discussion by unpezcado** [comfyanonymous/ComfyUI#5326] [Q&A]
Hello everyone!

When I want to open my workflow I get the following error, I haven't been able to find a solution

![Captura de pantalla 2024-10-22 102034](https://github.com/user-attachments/assets/4ce7eb32-cde6-4863-a92a-6810c4296459)

![Captura de pantalla 2024-10-22 102206](https://github.com/user-attachments/assets/70fc85a2-bdd8-4500-bf73-efdde646a945)

![Captura de pantalla 2024-10-22 131929](https://github.com/user-attachments/assets/fc343148-e40c-481d-b2b5-c03f3677e152)

I appreciate your help

**Comment by LukeG89** [comment]
You need to install the missing custom node, you can easily do it through the Manager:

![install](https://github.com/user-attachments/assets/efb271d9-baa4-4bbd-9a60-4b3e126baa4c)

**Reply by unpezcado** [reply]
Thanks @LukeG89  I already had node installed, this whole error happened yesterday when I updated ComfyUI

![Captura de pantalla 2024-10-22 135020](https://github.com/user-attachments/assets/f828f32c-0534-42ff-b6e1-a6eafe01eb0f)
![Captura de pantalla 2024-10-22 140937](https://github.com/user-attachments/assets/03d7339d-788a-44cf-a7aa-aad80fe882ce)
![Captura de pantalla 2024-10-22 141022](https://github.com/user-attachments/assets/24725ebc-9355-421f-a319-9d647b6ca498)

**Reply by ltdrdata** [reply]
Don't install ComfyUI-BiRefNet-ZHO and ComfyUI-BiRefNet redundantly.
They are conflicting.

**Reply by unpezcado** [reply]
Ok @ltdrdata, I'll try it... thks
